master/internal/core_experiment.go

```diff
@@ -319,6 +319,9 @@ func (m *Master) parseCreateExperiment(req *apiv1.CreateExperimentRequest, user
 	taskSpec := *m.taskSpec
 	taskSpec.TaskContainerDefaults = taskContainerDefaults
 	taskSpec.TaskContainerDefaults.MergeIntoExpConfig(&config)
+	if defaulted.RawEntrypoint == nil && (req.Unmanaged == nil || !*req.Unmanaged) {
```

Comment:
eh i mean.. may as well _not_ do this and require a dummy entrypoint for the test internal usage. i don't 'fake expconf' sticking at the eventual solution, anyway